packages/plugin-commands-outdated/src/outdated.ts

```diff
@@ -39,6 +39,7 @@ export function rcOptionsTypes () {
     ], allTypes),
     compatible: Boolean,
     table: Boolean,
+    format: ['json'],
```

Comment:
adding this, I also think this would be good to support `markdown` table format with `outdated` command. I would like to use `outdated` command's output with GitHub's issue, and it will be helpful to support `markdown` table format

Reply:
> If we support this feature, it may be better to merge table and format options. such as format: ['json', 'table', 'list']

sounds good. Regarding markdown, I am not sure how frequently that would be needed.

Reply:
I added the process to handle `table` and `list` format option in 5d82338. Should we deprecate `table` option in `outdated` command?

Reply:
As for the `markdown` table format, the only use case I can come up with is copying and pasting the output into a GitHub issue. I feel it is good to support this, but should we support this? (Of course this will be implemented in the other PR if we will do)